api/envoy/config/route/v3/route.proto

```diff
@@ -155,6 +156,11 @@ message RouteConfiguration {
   // For instance, if the metadata is intended for the Router filter,
   // the filter name should be specified as ``envoy.filters.http.router``.
   core.v3.Metadata metadata = 17;
+
+  // Specifies a collection of Formatter plugins that can be called from request_headers_to_add and
+  // response_headers_to_add operations.
+  // [#extension-category: envoy.formatter]
+  repeated core.v3.TypedExtensionConfig formatters = 18;
```

Comment:
to be honest, I am not sure if this is the right place to configure - I considered to put inside `HeaderValueOption` as well. I would like to hear what others think 🙏